.github/workflows/email.yml

```diff
@@ -0,0 +1,98 @@
+name: email
+
+on:
+  push:
+    branches:
+      - master
+      - main 
+      
+jobs:
+  
+  
+  send_email:
+    runs-on: ubuntu-latest
+  
+    steps:
+      - name: get env
+        run: env       
+      - name: get git events
+        run:  cat "$GITHUB_EVENT_PATH"
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"  
+        
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+          
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v32
+        with:
+          separator: ","
+         
+          
+      - name: Send mail  
+       
+        if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true  
+        uses: dawidd6/action-send-mail@v3
+        with:
+   
+    # Required mail server address if not connection_url:
+           server_address: ${{ secrets.SMTP_PROVIDER}}
+           server_port: 465
+    # Optional whether this connection use TLS (default is true if server_port is 465)
+           secure: true
+    # Optional (recommended): mail server username:
+           username:  ${{ secrets.MAIL_USERNAME}} 
+ 
+    # Optional (recommended) mail server password:
+           password: ${{secrets.MAIL_PASSWORD}}
+         
+           
+    # Required mail subject:
+           subject: Chapel Merge ${{ github.event.head_commit.message }}  
+           # Required recipients' addresses:
+           to: bhavani@hpe.com
```

Comment:
I will change the recipient address to chapel dev in a followup PR once I verify if this works as expected.